<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HxfzWidget</class>
 <widget class="QWidget" name="HxfzWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1670</width>
    <height>971</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabBar::tab{
	background-color:rgb(0, 255, 255);
}
#tab.QWidget{
	background-color:rgb(6, 6, 29);
}
#tab_2.QWidget{
	background-color:rgb(6, 6, 29);
}
#tab_3.QWidget{
	background-color:rgb(6, 6, 29);
}
#tab.QWidget{
	background-color:rgb(6, 6, 29);
}
#tab.QWidget{
	background-color:rgb(6, 6, 29);
}
#tab_4.QWidget{
	background-color:rgb(6, 6, 29);
}
#tab_5.QWidget{
	background-color:rgb(6, 6, 29);
}
QTabWidget::pane {
    border: 1px solid rgb(6, 6, 29);
}
</string>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>雨刮器</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>雾笛</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>窗加热</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>总动员警铃</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>航行信号灯</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
